src/components/StakingComparison.js

```diff
@@ -0,0 +1,176 @@
+import PoolGlyph from "../assets/staking/staking-glyph-token-wallet.svg"
+
+const GradientContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+  background: linear-gradient(
+    83.46deg,
+    rgba(127, 127, 213, 0.2) 7.03%,
+    rgba(138, 168, 231, 0.2) 52.42%,
+    rgba(145, 234, 228, 0.2) 98.77%
+  );
+  padding: 2rem;
+  margin-top: 4rem;
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    padding: 2rem 1.5rem;
+  }
+
+  h3 {
+    margin-top: 0;
+  }
+`
+
+const Flex = styled.div`
+  display: flex;
+  gap: 1.5rem;
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    flex-direction: column;
+  }
+`
+
+const Glyph = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  width: 3rem;
+  max-height: 3rem;
+`
+
+const StyledSoloGlyph = styled(SoloGlyph)`
+  path {
+    fill: ${({ theme }) => theme.colors.stakingGold};
+  }
+`
+const StyledSaasGlyph = styled(SaasGlyph)`
+  path {
+    fill: ${({ theme }) => theme.colors.stakingGreen};
+  }
+`
+const StyledPoolGlyph = styled(PoolGlyph)`
+  path {
+    fill: ${({ theme }) => theme.colors.stakingBlue};
+  }
+`
+
+const StakingComparison = ({ page, className }) => {
+  const intl = useIntl()
+  const themeContext = useContext(ThemeContext)
+  const { stakingGold, stakingGreen, stakingBlue } = themeContext.colors
+
+  const solo = {
+    title: "Solo staking",
+    linkText: "Learn more about solo staking",
+    to: "/staking/solo/",
+    matomo: {
+      eventCategory: `StakingComparison`,
+      eventAction: `Clicked`,
+      eventName: "clicked solo staking",
+    },
+    color: stakingGold,
+    glyph: <StyledSoloGlyph />,
+  }
+  const saas = {
+    title: "Staking as a service (SaaS)",
+    linkText: "Learn more about staking as a service",
+    to: "/staking/saas/",
+    matomo: {
+      eventCategory: `StakingComparison`,
+      eventAction: `Clicked`,
+      eventName: "clicked staking as a service",
+    },
+    color: stakingGreen,
+    glyph: <StyledSaasGlyph />,
+  }
+  const pools = {
+    title: "Pooled staking",
+    linkText: "Learn more about pooled staking",
+    to: "/staking/pools/",
+    matomo: {
+      eventCategory: `StakingComparison`,
+      eventAction: `Clicked`,
+      eventName: "clicked pooled staking",
+    },
+    color: stakingBlue,
+    glyph: <StyledPoolGlyph />,
+  }
+  const data = {
+    solo: [
+      {
+        ...saas,
+        content:
+          "With SaaS providers you're still required to deposit 32 ETH, but don't have to run hardware. You typically maintain access to your validator keys, but also need to share your signing keys so the operator can act on behalf of your validator. This introduces a layer of trust not present when running your own hardware, and unlike solo staking at home, SaaS does not help as much with geographic distribution of nodes. If you're uncomfortable operating hardware but still looking to stake 32 ETH...",
```

Comment:
> If you're uncomfortable operating hardware but still looking to stake 32 ETH...

I like this but am concerned it may not work well with translations given it's a sentence fragment. Perhaps we just finish the sentence?

> If you're uncomfortable operating hardware but still looking to stake 32 ETH, using a SaaS provider may be a good option for you.

Reply:
Ah yes... honestly was kinda left as a placeholder and meant to come back. Originally wanted to just include the link within the last sentence, but given this copy is in a data struct it wasn't working properly. Will just finish out the sentences per suggestion.